Driver API (#3564)
ROCm-Developer-Tools/HIP: docs/reference/driver_api_reference.rst

Reviewer: This file is only duplicating parts of the doxygen index https://rocm.docs.amd.com/projects/HIP/en/latest/doxygen/html/index.html, and that one is already listed in the table of contents (this one isn't, btw, this one is only listed in the index).

Author: Further discussion is required.

Reviewer 2: @MKKnorr  Check the HIP documentation planning document.

.. meta::
  :description: This chapter will present CUDA driver API porting and showcase equivalent operations in HIP.
  :keywords: AMD, ROCm, HIP, CUDA, driver API

.. _driver_api_reference:

*******************************************************************************
Driver API reference
*******************************************************************************

> On this page we present a list of driver API functions supported by HIP.

Reviewer 3: ```suggestion
This page presents a list of driver API functions supported by HIP.
```

> For comparison, compatibility and version information with the CUDA Driver API, please visit :doc:`hipify:index`.

Reviewer 3: ```suggestion
For comparison, compatibility, and version information with the CUDA Driver API, visit :doc:`hipify:index`.
```

Author: Updated

HIP driver API
==============

Context
-------

.. doxygengroup:: Context
   :content-only:

Module
------

.. doxygengroup:: Module
   :content-only:

> Driver Entry Point Access

Reviewer 3: ```suggestion
Driver entry point access
```
